app/views/users/_proposal.html.erb

```diff
@@ -0,0 +1,28 @@
+<tr id="proposal_<%= proposal.id %>">
+  <td>
+    <%= link_to proposal.title, proposal, proposal.retired? ? { class: 'retired' } : {} %>
+    <br>
+    <%= proposal.summary %>
+  </td>
+
+  <% if author?(proposal) %>
+    <td class="text-center">
+      <%= link_to t("users.proposals.send_notification"),
+                  new_proposal_notification_path(proposal_id: proposal.id),
+                  class: 'button hollow' %>
+    </td>
+  <% end %>
+
+  <% if author?(proposal) || proposal.retired? %>
```

Comment:
It took me a bit to understand all the conditionals around this block and realize there is only 2 scenarios. Maybe this could be easier to read:
```
<% if proposal.retired? %>
 <td class="text-center">
  <span class="label alert"><%= t('users.proposals.retired') %></span>
 </td>
<% elsif author?(proposal) %>
 <td class="text-center">
   <%= link_to t('users.proposals.retire'),
               retire_form_proposal_path(proposal),
               class: 'button hollow alert' %>
 </td>
<% end %>
```